src/styles/i-settings/_colors.scss

```diff
@@ -1,6 +1,6 @@
 /* === Neutrals === */
-$color-light: hsla(0, 0, 100, 1);
-$color-dark: hsla(0, 0, 0, 1);
+$color-light: hsla(0, 0%, 100%, 1);
+$color-dark: hsla(0, 0%, 0%, 1);
 
 /* === Palette === */
 $dark-purple: rgba(91, 20, 171, 1);
```

Comment:
We should make all of the color syntax match so let's change this rgba to hsl

Reply:
Good catch!
✅